src/librbd/crypto/Types.h

```diff
@@ -12,6 +19,28 @@ enum CipherMode {
     CIPHER_MODE_DEC,
 };
 
+struct SavedCryptoInterface {
```

Comment:
Nit: (just because it's not an interface)

```suggestion
struct ParentCryptoParams {
```